docs/deploy.rst

```diff
@@ -243,3 +243,75 @@ After everything starts, which may take a few minutes, you should be able
 to visit your Girder instance at ``http://X.X.X.X`` where ``X.X.X.X`` is the
 IP address in the container description above. Congratulations, you
 have a full Girder instance available on Google Container Engine!
+
+Elastic Beanstalk
+-----------------
+
+Girder comes with pre-packaged configurations for deploying onto Elastic Beanstalk's
+`Python platform <http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/concepts.platforms.html#concepts.platforms.python>`_
+(both 2.7 and 3.4).
+
+The configurations live within ``devops/beanstalk`` and are designed to be copied into your working Girder directory
+at deploy time.
+
+The following assumes you have a a checked out copy of Girder (using git) and an existing MongoDB instance which
+can be accessed by your Beanstalk application.
+
+.. note:: It is **highly** recommended to perform the following steps in an isolated virtual
+	  environment using pip. For more see the documentation for `Virtualenv <https://virtualenv.pypa.io/en/stable/>`_.
+
+From within the checked out copy of Girder, install and configure the CLI tools: ::
+
+  $ pip install awscli awsebcli
+  $ aws configure
+
+Initialize the Beanstalk application with a custom name. This is an interactive process
+that will ask various questions about your setup (see above for supported platforms): ::
+
+  $ eb init my-beanstalk-app
+
+Build Girder and its client-side assets locally: ::
+
+  $ pip install -e .[plugins]  # optionally build specific plugins
```

Comment:
You should clarify that plugins should be replaced with a list of plugins that are desired to be installed and have python dependencies.  This reads as commands you want people to be typing into their terminal, but "plugins" isn't something they'd ever type.

Maybe

```
Build Girder and its client-side assets locally: ::

When not installing any plugins with Python dependencies:
    
    $ pip install -e .

An example when installing a plugin with a Python dependency, in this case the `dicom_viewer`.  Use a comma separated list in the case of multiple plugins.

    $ pip install -e .[dicom_viewer]
```